clients/web/src/views/body/FrontPageView.js

```diff
@@ -37,16 +37,18 @@ var FrontPageView = View.extend({
         }
     },
 
-    initialize: function () {
+    initialize: function (settings) {
         cancelRestRequests('fetch');
+        this.brandName = settings.brandName;
```

Comment:
This should have a default, if `settings.brandName` is not provided. `'Girder'` would be fine.